CONVERTER-SERVICE/src/feature-convertor/feature-convertor.js

```diff
@@ -0,0 +1,38 @@
+const FfmpegCommand = require('fluent-ffmpeg');
+class VideoConvertor {
+    constructor(input, output, fps, finalSize){
+        this.input = input;
+        this.output = output;
+        this.fps = fps;
+        this.finalSize = finalSize;
+    }
+    conversor(){
+        let variable = false;
```

Comment:
I think "let variable = false;" is not necessary. At line 22, you can return false and you can also return true at line 28.